code/game/turfs/simulated/walls_reinforced.dm

```diff
@@ -34,6 +34,14 @@
 			if(WALLRODSCUT)
 				user << "It has no outer grille, external reinforced cover or external support rods and the inner reinforced cover is exposed" //And that's terrible
 
+//We need to export this here because we want to handle it differently
+//This took me longer to find this than it should havle
+/turf/simulated/wall/r_wall/relativewall()
+	if(d_state) //We are fucking building
+		return //Fuck off
+	var/junction = findSmoothingNeighbors()

```

Comment:
What do you mean ? If the reinforced wall has d_state (aka it's finished), we want it to smoothwall